packages/core/react-loosely-lazy/src/lazy-wait/__tests__/lazy-wait.integration.test.tsx

```diff
@@ -27,8 +22,7 @@ describe('LazyWait', () => {
     ['lazyForPaint', lazyForPaint],
     ['lazy', lazy],
   ])('%s', (_, lazyMethod) => {
-    // lazyForPaint uses useMemo, which has a side effect
-    const strictModeCalls = lazyMethod === lazyForPaint ? 1 : 0;
+    const strictModeCalls = 1;
```

Comment:
New implementation triggers double side effects on all types. We rely on the loader (webpack/parcel) to be smart. That's ok, right?

Reply:
Why does it double exactly?

Reply:
Because `useSubscription` uses `useMemo`, so it gets triggered twice for `lazy` too (the code paths between `lazyForPaint` and `lazy` are now the same) while before `lazy` used `useEffect`.